pkg/scheduler/cache/util.go

```diff
@@ -29,11 +32,51 @@ func CreateNodeNameToInfoMap(pods []*v1.Pod, nodes []*v1.Node) map[string]*NodeI
 		}
 		nodeNameToInfo[nodeName].AddPod(pod)
 	}
+	imageStatesMap := createNodeImageStatesMap(nodes)
+
 	for _, node := range nodes {
 		if _, ok := nodeNameToInfo[node.Name]; !ok {
 			nodeNameToInfo[node.Name] = NewNodeInfo()
 		}
-		nodeNameToInfo[node.Name].SetNode(node)
+		nodeInfo := nodeNameToInfo[node.Name]
+		nodeInfo.imageStates = imageStatesMap[node.Name]
+		nodeInfo.SetNode(node)
 	}
 	return nodeNameToInfo
 }
+
+func createNodeImageStatesMap(nodes []*v1.Node) map[string]map[string]*ImageStateSummary {
```

Comment:
Mind to add a brief comment to these two helpers as well?

Reply:
Done and I changed the other function's name to createImageExistenceMap.

Thanks for the comments!